examples/1_Introduction/02_intro_sandstone_parallel_roi.ipynb

```diff
@@ -46,7 +45,9 @@
     "- Load and investigate a real data set.\n",
     "- Determine geometric information of the data and set up CIL data structures.\n",
     "- Apply CIL processors to pre-process the data, including normalisation, negative log, region-of-interest and centre of rotation correction.\n",
-    "- Compute FBP reconstruction using CIL and compare with reconstruction provided."
+    "- Compute FBP reconstruction using CIL and compare with reconstruction provided.\n",
+    "\n",
+    "The data can be downloaded from: https://zenodo.org/record/4912435#.YvJ6tvfTXu0"
```

Comment:
As before provide the dataset download link (it makes it easier for us to check too)